README.md

```diff
@@ -213,15 +213,9 @@ what to put for each step.
 1. The elasticsearch instance domain can be one that is prexisting if
    used only for development.
 
-1. Next you will need a Google Client ID and Client secret. If you
-   missed it before, instructions to set this up are found on the
-   [Boardwalk deployment page](boardwalk/README.md#deployment).
-   _Boardwalk_ has functionality to export metadata to Broad's
-   FireCloud. In order to use it you need to provide your Google Cloud
-   Platform credentials.
-
-   You will also be asked for a Google site verfication code. For this
-   field you can just enter `NONE`.
+1. Provide the Google Client ID and Client Secret you got from the
+   OAuth2 app. You will also be asked for a Google site verfication
+   code. For this field you can just enter `NONE`.
```

Comment:
1. Please keep the part linking to the Boardwalk instructions. You just don't the part starting with `_Boardwalk_ has the functionality...`.
2. The Google site verification is useful for Google analytics. You should not say to `enter NONE`. You should say to enter it if you have it, and if you don't have one, to enter `NONE`.